defaults/main.yml

```diff
@@ -4,8 +4,16 @@
 monasca_checks: {}
 monasca_agent_system_only: False
 agent_reconfigure_script: /usr/local/bin/monasca-reconfigure
+monasca_agent_check_frequency: 60
+
+log_dir: /var/log
+monasca_log_dir: "{{ log_dir }}/monasca"
+monasca_agent_log_dir: "{{ monasca_log_dir }}/agent"
 
 skip_install: False
-monasca_virtualenv_dir: /opt/monasca
```

Comment:
The new name gives more flexibility. You may want to install agent in the separate virtualenv from the rest of monasca components.